components/publish/hooks/useTestimonyEmail.ts

```diff
@@ -39,12 +39,11 @@ export const useTestimonyEmail = () => {
     .filter(Boolean)
     .join("\n\n")
 
-  const plainBody = formatTestimonyPlaintext(markdownBody),
-    htmlBody = formatTestimony(markdownBody).__html
+  const htmlBody = formatTestimony(markdownBody).__html
 
   const mailToUrl = `mailto:${to}?subject=${encodeURIComponent(
     subject
-  )}&body=${encodeURIComponent(plainBody)}`
+  )}&body=${encodeURIComponent(markdownBody)}`
```

Comment:
`markdownBody` can include some formatting like 

```
**bold** [links](google.com)
```

So including it directly in the email will display the raw markup. Please take a look at https://github.com/alexjball/maple/commit/7b90813bccffa47b91645120bd3ca734e3314944 for how to update the plain text formatter to match newline formatting and link handling.

Reply:
Quick question, does the link handling work for plain text in your email client?